gtk2_ardour/po/fr.po

```diff
@@ -22617,9 +22670,6 @@ msgstr "Fichier vidéo d'entrée"
 #~ "\n"
 #~ "Nous allons configurer le logiciel avant que vous ne l'utilisiez.</span> "
 
-#~ msgid "GUI and Font scaling:"
```

Comment:
Just curious about the removals starting from here.

Reply:
They are some old translations (see the "#") which are conserved here for historical/ease reason.
When I can be sure that one can be removed (ex: it's already re-translated above), it's better to remove the "duplicated one".